### PR TITLE
update parl.maddpg without import gym

### PR DESCRIPTION
**Pull request by rical730**

### CODE REVIEW

File: parl/algorithms/fluid/maddpg.py [hunk 1]
```diff
@@ -22,13 +22,13 @@
 
 __all__ = ['MADDPG']
 
-from gym import spaces
 from parl.core.fluid.policy_distribution import SoftCategoricalDistribution
 from parl.core.fluid.policy_distribution import SoftMultiCategoricalDistribution
 
 
 def SoftPDistribution(logits, act_space):
```
Comment (TomorrowIsAnOtherDay) [inline parl/algorithms/fluid/maddpg.py:29]: Could you add some comments about this function?

File: README.cn.md [hunk 2]
```diff
@@ -82,6 +82,6 @@ pip install parl
 - [冠军解决方案：NIPS2018强化学习假肢挑战赛](examples/NeurIPS2018-AI-for-Prosthetics-Challenge/)
 - [冠军解决方案：NIPS2019强化学习仿生人控制赛事](examples/NeurIPS2019-Learn-to-Move-Challenge/)
 
-<img src=".github/NeurlIPS2018.gif" width = "300" height ="200" alt="NeurlIPS2018"/> <img src=".github/Half-Cheetah.gif" width = "300" height ="200" alt="Half-Cheetah"/> <img src=".github/Breakout.gif" width = "200" height ="200" alt="Breakout"/> 
+<img src=".github/NeurlIPS2019.gif" width = "300" height ="200" alt="NeurlIPS2018"/> <img src=".github/Half-Cheetah.gif" width = "300" height ="200" alt="Half-Cheetah"/> <img src=".github/Breakout.gif" width = "200" height ="200" alt="Breakout"/>
```
Comment (TomorrowIsAnOtherDay) [inline README.cn.md]: I found that we had this image uploaded before. We can reuse it here. 
https://github.com/PaddlePaddle/PARL/tree/develop/examples/NeurIPS2019-Learn-to-Move-Challenge/image

File: parl/algorithms/fluid/maddpg.py (hunk 1 above)
Comment (rical730) [reply parl/algorithms/fluid/maddpg.py:29]: done

File: README.cn.md (hunk 2 above)
Comment (rical730) [reply README.cn.md]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
Looks good to me.

**Pull request merged**